snippets/stringToArrayOfWords.md

```diff
@@ -0,0 +1,12 @@
+### stringToArrayOfWords
+
+Converts a given string into an array of words
+
+Use `String.split(/[\W_]+/)` to convert the string to an array with the use of regex.
```

Comment:
The pattern is no able to be user supplied, so `String.split` would suffice here